roles/marathon_chronos/templates/chronos-consul.j2

```diff
@@ -0,0 +1,12 @@
+{
+ "service": {
+   "name": "chronos",
+   "tags": [ "chronos" ],
+   "port": 14400,
+   "check": {
+     "script": "curl --silent --show-error --fail --dump-header /dev/stderr --retry 2 --connect-timeout 5 --max-time 5 http://chronos.service.consul:14400/ping",
```

Comment:
I've been investigating consul health checks... I totally agree. updating the UI would be simplier and more logical